src/plugins/workspace/public/components/workspace_form/workspace_collaborator_table.tsx

```diff
@@ -190,13 +233,21 @@ export const WorkspaceCollaboratorTable = ({
           setSelection([]);
           modal.close();
         },
+        selections: selection,
       });
     };
 
     return (
-      <EuiButton color="danger" iconType="trash" onClick={onClick}>
+      <EuiButton
+        color="danger"
+        iconType="trash"
+        onClick={onClick}
+        data-test-subj="confirm-delete-button"
+      >
         {i18n.translate('workspace.detail.collaborator.delete', {
-          defaultMessage: 'Delete {num} collaborators',
+          defaultMessage: `Delete {num} ${
+            selection.length === 1 ? 'collaborator' : 'collaborators'
```

Comment:
According to https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8398/files, lets avoid dynamic uses of i18n. 
You could try to follow this way.
```
              title: i18n.translate('workspace.detail.collaborator.delete.success', {
                defaultMessage: 'Delete collaborator{pluralSuffix} successfully.',
                values: {
                  pluralSuffix: selection.length > 1 ? 's' : '',
                },
              }),
```

Reply:
Thanks for the reviews, already updated

Reply:
i18 provide a native way for pluralization https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/packages/osd-i18n/GUIDELINE.md?plain=1#L69-L76